### PR TITLE
Delete site API v2

### PR DESCRIPTION
**Pull request by jmendeza**
https://github.com/craftercms/craftercms/issues/6061
Delete site API v2

### CODE REVIEW

File: src/main/resources/crafter/studio/database/createDDL.sql [hunk 1]
```diff
@@ -98,14 +98,51 @@ BEGIN
...
+
 CREATE TABLE _meta (
   `version` VARCHAR(10) NOT NULL,
   `integrity` BIGINT(10),
   `studio_id` VARCHAR(40) NOT NULL,
   PRIMARY KEY (`version`)
 ) ;
 
-INSERT INTO _meta (version, studio_id) VALUES ('4.0.1.16', UUID()) ;
+INSERT INTO _meta (version, studio_id) VALUES ('4.1.3', UUID()) ;
```
Comment (sumerjabri) [inline src/main/resources/crafter/studio/database/createDDL.sql]: 4.2.0?

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml [hunk 2]
```diff
@@ -531,6 +531,11 @@ pipelines:
               - src: repo-bootstrap/global/configuration/samples/sample-studio-config-override.yaml
                 dest: /configuration/samples/sample-studio-config-override.yaml
             commitDetails: Update sample studio override config file
+      - currentVersion: 4.1.2
+        nextVersion: 4.1.3
```
Comment (sumerjabri) [inline src/main/resources/crafter/studio/upgrade/pipelines.yaml]: 4.2.0?

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Please make DRAFT until 4.1.0 is out and tagged. Please also fix the issues noted.

File: src/main/resources/crafter/studio/database/createDDL.sql (hunk 1 above)
Comment (jmendeza) [reply src/main/resources/crafter/studio/database/createDDL.sql]: Done

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml (hunk 2 above)
Comment (jmendeza) [reply src/main/resources/crafter/studio/upgrade/pipelines.yaml]: Done

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**